EIPS/eip-4527.md

```diff
@@ -4,17 +4,20 @@ title: QR Code data transmission protocol for the offline wallets
 description: QR Code data transmission protocol between wallets and offline signers.
 author: Aaron Chen (@aaronisme), Sora Lee (@soralit), ligi (@ligi), Dan Miller (@danjm), AndreasGassmann (@andreasgassmann), xardass (@xardass), Lixin Liu (@BitcoinLixin)
 discussions-to: https://ethereum-magicians.org/t/add-qr-code-scanning-between-software-wallet-cold-signer-hardware-wallet/6568
-status: Stagnant
+status: Draft 
```

Comment:
```suggestion
status: Draft
```
This is a small typo, but it blocks this PR.

Reply:
Hi @Pandapip1 I have updated the file, would you please check it again?